### ISSUE TITLE
Add IM event Yaml and Chip tool testing

### ISSUE THREAD

**Issue by yunhanw-google** [project-chip/connectedhomeip#12460]
#### Problem
pending PR for #12059, no example integration for IM eventing so far. Once it is ready, it is good to test with TestCluster where we can use command to instruct server to generate event, then call readEvent to retrieve events, for other real cluster, like diagnostic cluster or basic application cluster, their event generation is related to platform, which is a little hard to achieve in yaml automation test.

**Comment by vivien-apple** [comment]
Duplicate of #11343
